vllm/model_executor/layers/attention.py

```diff
@@ -220,6 +220,8 @@ def _paged_attention(
 ) -> torch.Tensor:
     output = torch.empty_like(query)
 
+    enable_fp8_kv_cache = key_cache.dtype == torch.uint8
+
```

Comment:
Would this invalidate 8bit KV cache other than FP8, unnecessarily?

Reply:
+1 Can we get this from model config?